### ISSUE TITLE
Yoda condition false positive

### ISSUE THREAD

**Issue by GaryJones** [WordPress-Coding-Standards/WordPress-Coding-Standards#1181]
After the following WP-CLI code:

```php
 function is_windows() {
	return false !== ( $test_is_windows = getenv( 'WP_CLI_TEST_IS_WINDOWS' ) ) ? (bool) $test_is_windows : strtoupper( substr( PHP_OS, 0, 3 ) ) === 'WIN';
 }
```

...was changed to:

```php
 function is_windows() {
	return false !== ( $test_is_windows = getenv( 'WP_CLI_TEST_IS_WINDOWS' ) ) ? (bool) $test_is_windows : 0 === stripos( PHP_OS, 'WIN' );
 }
```

...PHPCS complained that the `return` line was not using Yoda conditions. This seems like a false positive, probably from the number of `=` symbols present.

**Comment by jrfnl** [comment]
That false positive should be looked into, however, the sniff does not contain a fixer and no matter what, wouldn't fix code to be something completely different - `strtoupper( substr( PHP_OS, 0, 3 ) ) === 'WIN'` vs `0 === stripos( PHP_OS, 'WIN' )` -.

In the mean time, for the love of code, can someone refactor that line to a readable code block ?

**Comment by GaryJones** [comment]
> In the mean time, for the love of code, can someone refactor that line to a readable code block ?

https://github.com/wp-cli/wp-cli/pull/4417/files#diff-e02dc22fa7c2dd0a3059084ee686ec10R544 :-)
